data/common.yaml

```diff
@@ -58,6 +58,7 @@ mcollective::server: true
 mcollective::client: false
 mcollective::purge: true
 mcollective::facts_refresh_interval: 10
+mcollective::periodic_runmode: "cron"
```

Comment:
With the default value at the level of the parameter, no need to put it in hiera.

Reply:
> Thanks, I think it is better to recreate the branch and force-push it to avoid unneeded commits in data/common.yaml first adding this unneeded value and then removing it.

You can rewrite history to get rid of the details (also called squashing your commits).

From your working directory:

```
git rebase -i origin/main # Interactively rewrite history
```

This will bring your editor listing all commits in your branch.  Keep the first line unchanged, and replace `pick` with `squash` for all subsequent lines.  Save and quit, your editor will pop again and allow you to combine all the commit messages in a single one.

Then push your updated branch:

```
git push -f            # Send the changes (-f is required because we re-wrote history)
```